third_party/move/move-model/src/ast.rs

```diff
@@ -2993,3 +3091,183 @@ impl<'a> fmt::Display for EnvDisplay<'a, Spec> {
         Ok(())
     }
 }
+
+#[test]
```

Comment:
nit: it is more idiomatic (and helpful in the presence of further changes and maintainance) to wrap all this with:
```
#[cfg(test)]
mod tests {
    // code to be wrapped
}
```

Source: https://doc.rust-lang.org/book/ch11-01-writing-tests.html#the-anatomy-of-a-test-function

Reply:
I found I had to add several `use` statements as well, then I read that page and saw an example using `use super::*;`, but after I pushed my new code.  I suppose explicit `use` statements provides guidance to users of the code.